tests/framework/validators/EmailValidatorTest.php

```diff
@@ -26,8 +26,10 @@ public function testValidateValue()
         $this->assertFalse($validator->validate('Carsten Brandt <mail@cebe.cc>'));
         $this->assertFalse($validator->validate('"Carsten Brandt" <mail@cebe.cc>'));
         $this->assertFalse($validator->validate('<mail@cebe.cc>'));
-        $this->assertFalse($validator->validate('info@örtliches.de'));
-        $this->assertFalse($validator->validate('sam@рмкреатиф.ru'));
+        $this->assertTrue($validator->validate('info@örtliches.de'));
+        $this->assertTrue($validator->validate('sam@рмкреатиф.ru'));
```

Comment:
@samdark yeah, you're right, but info@örtliches.de according to validator is valid email. but anyway will move it to IDN stuff only, thanks